daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala

```diff
@@ -592,7 +592,9 @@ object Main {
     try {
       val compiler = Compiler()
       val processor = Timer.getResult("reloading", compiler.reload(savedParser))
-      displayDiagnostics(processor)
```

Comment:
I'm was thinking it might be useful output diagnostics if the -v option is added. But on second thought, probably not--you shouldn't need to enable parse/unparse verbosity just to get information about a saved parser. Probably the right thing is something like a "parser-info" command that can print out metadata stored inside a saved parser. That could include things like daffodil version (which we already have), java version, date/time built, diagnostics, schema information. But that feels like very low priority, especially since most of this information isn't even available at the moment.